var/spack/repos/builtin/packages/py-mongo/package.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyMongo(PythonPackage):
+    """Python driver for MongoDB <http://www.mongodb.org>"""
+
+    homepage = "http://github.com/mongodb/mongo-python-driver"
+    url      = "https://pypi.io/packages/source/p/pymongo/pymongo-3.3.0.tar.gz"
+
+    version('3.3.0', '42cd12a5014fb7d3e1987ca04f5c651f')
```

Comment:
A couple things worth adding:
```python
depends_on('python@2.6:2.8,3.3:')
```
PyMongo only supports these versions of Python.
```python
depends_on('py-setuptools', type='build')
```
This isn't required, but if it's present, the setup.py will use setuptools to handle the installation.

Since this is already merged, you'll need to submit a separate PR to add these.